runtime/compiler/runtime/ClassUnloadAssumption.cpp

```diff
@@ -499,14 +496,13 @@ void TR_RuntimeAssumptionTable::markAssumptionsAndDetach(void * md, bool reclaim
             #if defined(PROD_WITH_ASSUMES) || defined(DEBUG)
             TR_RuntimeAssumptionKind kind = cursor->getAssumptionKind();
             TR_ASSERT(kind == RuntimeAssumptionOnClassRedefinitionPIC ||
-                      kind == RuntimeAssumptionOnClassRedefinitionUPIC ||
+                      kind == RuntimeAssumptionOnClassRedefinitionUPIC || 
```

Comment:
Please remove the blank at the end of this line.  It is causing the "Line Endings Check" error that is being reported on this file.